lib/ukallocbbuddy/bbuddy.c

```diff
@@ -81,6 +86,10 @@ struct uk_bbpalloc {
 	chunk_head_t *free_head[FREELIST_SIZE];
 	chunk_head_t free_tail[FREELIST_SIZE];
 	struct uk_bbpalloc_memr *memr_head;
+#ifdef CONFIG_LIBUKALLOCBBUDDY_SMP_SAFE
```

Comment:
Remove the config flag check; this is because on the uniprocessor system, the uk_spinlock is mapped to an empty structure hence required memory is 0 bytes.
```suggestion
```